### ISSUE TITLE
[Feature Request] Choose which OptiFine installation to overwrite.

### ISSUE THREAD

**Issue by HydronicDev** [sp614x/optifine#4751]
Currently, I have multiple versions of OptiFine installed. When I update from 1.16.1 to 1.16.2, it changes the 1.12 installation instead.  This happens every update, as far back as I can remember. It would be nice to implement some kind of GUI, or even a startup flag you could include to say which installation to write to. As of now, I have to go into the wrong one, fix it, and then change the one it should've installed to manually.

**Comment by Docteh** [comment]
What is the name of the instance of your 1.12 installation? I thought that the Installer just overwrites an instance called "OptiFine"

**Comment by HydronicDev** [comment]
It's called "OptiFine 1.12.2"

**Comment by sp614x** [comment]
The installer overwrites the instance called OptiFine.

**Comment by Docteh** [comment]
I found what they mean.

I renamed my OptiFine instance to "OptiFine 1.16.2" and then installed optifine for 1.12.2, and now my "OptiFine 1.16.2" instance has 1.12.2
![image](https://user-images.githubusercontent.com/190571/91920645-9222a580-ec7e-11ea-823a-94f23b4cdb04.png)

Profiles created in the launcher have randomly generated keys (301...)
```json
    "301135f20b4a02d7218ffb25e35e7ad5" : {
      "created" : "2020-09-02T01:14:18.895Z",
      "icon" : "Furnace",
      "lastUsed" : "1970-01-01T00:00:00.000Z",
      "lastVersionId" : "1.16.2-OptiFine_HD_U_G3",
      "name" : "OptiFine",
      "type" : "custom"
    },
```

While the profile effected by the OptiFine installer has a key of "OptiFine", and renaming the profile to "OptiFine 1.12.2" in the launcher doesn't change this. 

`.minecraft/launcher_profiles.json`

@HydronicDev duplicate the profile, and use that one for 1.12.2 going forward. Maybe just rename the other one back to "OptiFine" and then just leave it be.
